pt-BR/active_job_basics.md

```diff
@@ -256,8 +256,8 @@ end
 ProcessVideoJob.perform_later(Video.last)
 ```
 
-NOTE: Make sure your queuing backend "listens" on your queue name. For some
-backends you need to specify the queues to listen to.
+NOTE: Tenha certeza de que o seu *backend* da fila "escuta" o nome da fila.
```

Comment:
```suggestion
NOTE: Tenha certeza de que o seu *backend* de fila "escuta" o nome da fila.
```

Acho que aqui fica mais claro se for um "de fila" por causa da palavra "queuing" e pra não misturar com a fila da frente, o que acha?

Reply:
Done.